spec/features/admin_views_list_of_users_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-feature 'admin views admin panel', %Q{
+feature 'admin views list of users', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.